microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -0,0 +1,336 @@
+
+1. In Solution Explorer, expand the **webview2_sample_uwp** project, and then select **MainPage.xaml.cs**.
+
+1. Below the `MainPage` constructor, add the following method, to pass the WinRT object from the native side of the host app to the web side of the host app:
+
+   ```csharp
+   private async void InitializeWebView2Async()
+   {
+      await WebView2.EnsureCoreWebView2Async();
+      var dispatchAdapter = new WinRTAdapter.DispatchAdapter();
+      WebView2.CoreWebView2.AddHostObjectToScript("Windows", dispatchAdapter.WrapNamedObject("Windows", dispatchAdapter));
+   }
+   ```
+    
+1. In the `MainPage` constructor, above the `StatusUpdate("Ready");` line, add the following code:
+
+   ```csharp
+   InitializeWebView2Async();
+   ```
+
+1. Right-click the **webview2_sample_uwp** project, and then select **Set as startup project**.
+
+1. Press `Shift+Ctrl+S` to save all files.
+
+1. Press `F5` to run the sample app.  The **WebView2 WinUI 2 UWP Sample** window opens.
+
+   ![The WebView2 WinUI 2 UWP Sample window.](winrt-from-js-images/webview2-winui-2-uwp-sample-app-window.png)
+
+The host app's web-side code, including DevTools **Console**, can now access properties and methods of the specified namespaces or classes of the host object.
+
+
+<!-- =============================================== -->
+## Step 9. Call properties and methods on the host object from web-side JavaScript
+
+Next, use the DevTools **Console** to demonstrate that web-side code can potentially call the included, specified host-side API items.
+
+1. Click in the main part of the WebView2 sample app window to give it focus, and then press `Ctrl+Shift+I` to open Microsoft Edge DevTools.  Or, right-click the page, and then select **Inspect**.  The Microsoft Edge DevTools window opens.
+
+1. If the Microsoft Edge DevTools window isn't visible, press `Alt+Tab` to display the DevTools window.
+
+1. In the **Console** of DevTools, paste the following code:
+
+   ```javascript
+   (new chrome.webview.hostObjects.sync.Windows.Globalization.Language("en-US")).displayName
+   ```
+
+   The **Console** outputs `English (United States)`, or another language, demonstrating that your app's host-side code can be called from web-side code.
+
+   ![Using the DevTools Console to test calling native-side code from web-side code.](winrt-from-js-images/devtools-console-calling-native-side-code.png)
+
+1. Close the UWP sample app.  The DevTools window also closes.
+
+
+<!-- ====================================================================== -->
+## WebView2 Properties available in the WinRTAdapter Property Pages
+
+This section is for reference.
+
+![Properties that are listed in the WinRTAdapter Property Pages.](winrt-from-js-images/winrtadapter-property-pages.png)
+
+For help about a property, click a property row.  Help is shown at the bottom of the dialog.  Command-line help also contains similar information.
+
+| WebView2 property | Description |
```

Comment:
@jasonstephen15 @david-risney - Do we need the section about the properties? Delete the half-empty table, or fill it in, dup'ing the dialog's UI/help?

Rendered: https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/winrt-from-js?branch=pr-en-us-1910#webview2-properties-available-in-the-winrtadapter-property-pages

There's no point in having the table unless it has verbose content, like maybe from command line.